src/filters/KalmanFilter.h

```diff
@@ -0,0 +1,278 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <Eigen/LU>
+#include <unsupported/Eigen/MatrixFunctions>
```

Comment:
The unsupported refers to the fact that it's a "contribution from various users", but it still ships with the Eigen installation, so no additional configuration is required. It's used because in order to convert the system from continuous to discrete and back, we need to take the matrix logarithm and exponential, which are implemented in the `MatrixFunctions` module. Is that ok?